climada/hazard/trop_cyclone/trop_cyclone_windfields.py

```diff
@@ -1,53 +1,22 @@
-"""
-This file is part of CLIMADA.
-
-Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
-
-CLIMADA is free software: you can redistribute it and/or modify it under the
-terms of the GNU General Public License as published by the Free
-Software Foundation, version 3.
-
-CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
-WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-PARTICULAR PURPOSE.  See the GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License along
-with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
-
----
-
-Define TC wind hazard (TropCyclone class).
-"""
-
-__all__ = ['TropCyclone']
-
-import copy
-import datetime as dt
-import itertools
 import logging
```

Comment:
Please add a module docstring to this file (or rather: revert the deletion of it)

Reply:
This was unintenionally not added to the newly created second file. I will add it now